salt/utils/thin.py

```diff
@@ -453,16 +451,16 @@ def get_tops(extra_mods="", so_mods=""):
                 else:
                     tops.append(os.path.join(moddir, base + ".py"))
             except ImportError as err:
-                log.exception(err)
-                log.error('Unable to import extra-module "%s"', mod)
+                log.error(
+                    'Unable to import extra-module "%s": %s', mod, err, exc_info=True
+                )
```

Comment:
**question** should this be exception rather than error?

Reply:
(note: there are several of this same question)

Reply:
I opted to maintain the same log level, while including the exception traceback using `exc_info=True`